utils/docker/Dockerfile.focal

```diff
@@ -7,12 +7,12 @@ ARG DOCKER_IMAGE_NAME_TEMPLATE="mcr.microsoft.com/playwright/java:v%version%-foc
 # === INSTALL JDK and Maven ===
 
 RUN apt-get update && \
-    # Install install jdk 17 in a separate apt-get command so that
+    # Install install jdk 21 in a separate apt-get command so that
     # installing maven doesn't bring in jdk 11
-    apt-get install -y --no-install-recommends openjdk-17-jdk && \
+    apt-get install -y --no-install-recommends openjdk-21-jdk && \
```

Comment:
strictly speaking this comment in line 11 seems outdated and we can have a single apt-get install now.

Reply:
Good catch. Updated the scripts.